bin-ranges-transfer/src/main/java/uk/gov/pay/java_lambdas/bin_ranges_transfer/model/Version.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.java_lambdas.bin_ranges_transfer.model;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public enum Version {
+    V03, V04, UNKNOWN;
+    
+    private static final List<Version> supportedVersions = Arrays.asList(V03, V04);
+    private static final Logger logger = LoggerFactory.getLogger(Version.class);
+    
+    public static Version fromEnvironment(String versionStr) {
+        for (Version v : supportedVersions) {
+            if (v.name().equals(versionStr.toUpperCase())) {
```

Comment:
Does this actually need to be case-insensitive?

Reply:
not strictly, just some QOL incase a config gets fat fingered